.travis.yml

```diff
@@ -49,19 +49,12 @@ jobs:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
     - stage: "Additional Tests"
-      env: EMBER_TRY_SCENARIO=ember-lts-2.18
-    - env: EMBER_TRY_SCENARIO=ember-lts-3.4
+      env: EMBER_TRY_SCENARIO=ember-lts-3.4
+    - env: EMBER_TRY_SCENARIO=ember-lts-3.8
     - env: EMBER_TRY_SCENARIO=ember-release
     - env: EMBER_TRY_SCENARIO=ember-beta
     - env: EMBER_TRY_SCENARIO=ember-canary
     - env: EMBER_TRY_SCENARIO=ember-default-with-jquery
 
```

Comment:
Lost the yarn parts of .travis.yml here

Reply:
@kategengler do you think it is needed? I see that it works even without it.

Reply:
Yes. The before_install is probably unnecessary, it may only make sure you get the latest yarn version. The install step here is important because it adds `--no-lockfile` option that ensures dependencies float, without it your 'Floating Dependencies' stage is the same as part of the 'Tests' stage (see https://travis-ci.org/jelhan/ember-style-modifier/jobs/620199516#L480).

Reply:
Good to know! Thanks! Returned that bit to the travis config.